tap_mailchimp/schemas/list_members.json

```diff
@@ -210,6 +269,63 @@
         "null",
         "string"
       ]
+    },
+    "marketing_permissions": {
```

Comment:
I can see `marketing_permissions` as list of objects. Can you please verify the schema is expected? Reference: https://mailchimp.com/developer/marketing/api/list-members/list-members-info/

Reply:
Updated the datatype to Object[ ]